lib/components/note-info-item/index.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { FunctionComponent } from 'react';
+
+import type * as T from '../../types';
+
+type Props = {
+  noteDetails?: String | FunctionComponent | null;
+  noteName?: String;
```

Comment:
these props are already namespaces by means of being props for this component. you can see my other comment questioning the value of creating this component, but if we do then `details` and `name` would be sufficient unless you have a reason they need to be prefixed. it's awkward below when calling it since `noteName` and `noteDetails` don't stick out as being related to what it's used for.